src/Microsoft.DotNet.Interactive/Microsoft.DotNet.Interactive.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
I thought we were multi targeting here

Reply:
@colombod , we will be, I wanted to run all of the tests against the netstandard2.0 assembly, to ensure nothing was broken.

Reply:
We were discussing whether running tests against both netstandard2.0 and netstandard2.1 is worth the trouble. The netstandard2.0 version will be for an internal-only prototype and not supported long-term, so I'm inclined to just fix bugs if/when they occur rather than add to the maintenance burden.